### PR TITLE
link to Apache Harmony GitHub repository

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -10,8 +10,15 @@ source code of the Oracle JDK or OpenJDK!***
...
-The only existing JDK source code that we can look at is the dead Apache
-Harmony project.
+Non Compatible license:
+
+* Android libcore (GPL)
+* OpenJDK (GPL)
+
+Compatible license source code:
+
+* [scala-js](https://github.com/scala-js/scala-js/tree/master/javalib/src/main/scala/java) (Scala License <=> BSD 3-clause)
```
Comment (sjrd) [inline docs/contributing.md]: It's supposed to be written `Scala.js` ;-)

Comment (MasseGuillaume) [inline docs/contributing.md]: Would it be possible to create a GitHub repository to share javalib source code?

Comment (sjrd) [inline docs/contributing.md]: Concrete suggestions on how to achieve that are welcome.

**Comment by densh** [comment]
LGTM, thanks!

**Pull request merged**